frontend/src/components/user/UserCreateModal.vue

```diff
@@ -114,21 +135,10 @@ onMounted(() => {
             placeholder="Search by first name, last name, or email"
             class="col-12 pl-5"
             @update:model-value="searchIdirUsers"
+            autofocus
```

Comment:
Github warning: Attribute "autofocus" should go before "https://github.com/update:model-value"